getting_started/workflow/assets/escn_exporter/mesh.rst

```diff
@@ -0,0 +1,23 @@
+Mesh
+====
+
+Modifiers
+---------
+In the exporter there is an option :code:`Apply Modifiers` to
+control if mesh modifiers are applied to the exported mesh. 
+
+
+Shape Key
+---------
+Mesh shape key is also supported, however exporting each shape key
+is almost like exporting the mesh again, so don't be surprisied
+it takes a relatively long time.
+
+.. warning::
+    a lot of modifiers are not compatible with shape keys
```

Comment:
Capitalization of the first word, and the word incompatible is misspelled. 
Also, referring to it as "our" issues list is not typically considered good scientific writing.